src/components/MemberContainer.js

```diff
@@ -25,7 +25,7 @@ export default function Member(props) {
   const {member, isOdd, shape, ...rest} = props
   const colorName = member.color || 'blue'
   const color = themeGet(`colors.${colorName}.4`)(rest)
-  const avatarImage = hover ? member.gif : member.avatar
+  const avatarImage = hover ? member.gif : `https://avatars.githubusercontent.com/${member.handle}?s=600`
```

Comment:
💥 Nice. Do we always want this to be the GitHub avatar though? I think some of the team may have been uploading custom images intentionally for this in the past. Not sure though... cc. @yaili

Reply:
is this for the about page? If so I'd say yes, give folks the chance to choose.